tcms/xmlrpc/tests/test_testcaserun.py

```diff
@@ -37,10 +37,10 @@ def _fixture_setup(self):
         self.plan = TestPlanFactory(author=self.api_user, product=self.product)
         self.test_run = TestRunFactory(product_version=self.version, build=self.build,
                                        default_tester=None, plan=self.plan)
-        self.status = TestCaseRunStatus.objects.get(name='IDLE')
+        self.status = TestExecutionStatus.objects.get(name='IDLE')
         self.case = TestCaseFactory(author=self.api_user, default_tester=None, plan=[self.plan])
 
-        self.case_run_pks = []
+        self.execution_pks = []
 
     def test_create_with_no_required_fields(self):
```

Comment:
Inside tests where you have `exec.TestCaseRun.` Try to mix between the new and the old names, that is if you have several scenarios executing the same RPC method you should make sure at least one of them uses the new names and 1 of them uses the old names. This will verify the backwards & forwards compatibility until we have time to upgrade the API client library and all the plugins calling these methods (at which point we'll update all the references in the test suite as well).